saleor/settings.py

```diff
@@ -471,7 +472,16 @@ def get_url_from_env(name, *, schemes=None) -> str | None:
 # Following the recommendation of https://tools.ietf.org/html/rfc5322#section-2.1.1
 DEFAULT_MAX_EMAIL_DISPLAY_NAME_LENGTH = 78
 
-COUNTRIES_OVERRIDE = {"EU": "European Union"}
+COUNTRIES_OVERRIDE = {
+    "EU": "European Union",
+    "XK": {
+        "name": _("Kosovo"),
+        "alpha3": "XXK",
+        "ioc_code": "KOS",
+        "numeric": "383",
+        "numeric_padded": "0383",
```

Comment:
Django countries has also option to add `flag` https://github.com/SmileyChris/django-countries?tab=readme-ov-file#the-country-object:
> flag
Contains a URL to the flag. 

Do we want to add it as well?